todo_app/admin/user_requests.py

```diff
@@ -50,15 +54,36 @@ def add_user(self, user: User) -> None:
 
     def get_user_role(self, user: User) -> int:
         if os.getenv("LOGIN_DISABLED") == "True":
+            app.logger.critical("Login disabled - this should never happen!!")
```

Comment:
Nice, this is a really simple but useful log that we could easily set up an alert to watch out for!
It might push us to move this into a function so that whenever we call our `is_login_disabled()` function we know it will always trigger this log